packages/react-aria-components/docs/react-aria-components.mdx

```diff
@@ -0,0 +1,681 @@
+import DatePicker from '@react-spectrum/docs/pages/assets/component-illustrations/DatePicker.svg';
+import DateRangePicker from '@react-spectrum/docs/pages/assets/component-illustrations/DateRangePicker.svg';
+
+---
+category: Concepts
+keywords: [aria, accessibility, react, component]
+navigationTitle: Overview
+type: component
+after_version: 1.0.0
+description: React Aria Components is a library of unstyled components built on top of the React Aria hooks. It provides a simpler way to build accessible components with custom styles, while offering the flexibility to drop down to hooks for even more customizability where needed.
+---
+
+# React Aria Components
+
+React Aria Components is a library of unstyled components built on top of the React Aria hooks. It provides a simpler way to build accessible components with custom styles, while offering the flexibility to drop down to hooks for even more customizability where needed.
+
+## What is React Aria Components?
+
+React Aria Components is a new library of unstyled components implementing [ARIA patterns](https://www.w3.org/WAI/ARIA/apg/), built on top of the existing [React Aria](index.html) hooks. It provides components for common UI patterns, with [accessibility](accessibility.html), [internationalization](internationalization.html), [interactions](interactions.html), and behavior built in, allowing you to focus on your unique design and styling rather than re-building these challenging aspects. React Aria has been meticulously tested across a wide variety of devices, interaction modalities, and assistive technologies to ensure the best experience possible for all users.
+
+Compared with the React Aria hooks, the components provide a default DOM structure and styling API, and abstract away the glue code necessary to connect the hooks together. The components and hooks also work together, allowing them to be mixed and matched depending on the level of customization required. Eventually, React Aria Components will be recommended as a starting point when building a new component, dropping down to hooks only when additional flexibility is needed. Both hooks and components will continue to be maintained and developed going forward.
+
+## Status
+
+React Aria Components is currently in **alpha**. This means APIs will likely change in future updates as we discover the best ways to use it, and there are some known bugs and limitations. That said, it is based on a solid and battle-tested foundation in React Aria, and we would love for you to try it out and give us feedback! This will directly help us shape the APIs and make it the best library it can be. Please report issues and feature requests [on GitHub](https://github.com/adobe/react-spectrum/issues).
+
+## Installation
+
+React Aria Components can be installed using a package manager like [npm](https://docs.npmjs.com/cli/npm) or
+[yarn](https://classic.yarnpkg.com/lang/en/).
+
+```
+yarn add react-aria-components
```

Comment:
Will this be version 1 alpha? Or version 3? Do we need to warn them that the 0.4.0 version exists and isn't this project?

Reply:
Yeah 1.0.0-alpha.0 should get automatically installed as the latest version.